UPGRADE.md

```diff
@@ -512,3 +512,10 @@ The back end widgets `pageSelector` and `fileSelector` have been removed. Use th
 ### Public folder
 
 The public folder is now called `public` by default. It can be renamed in the `composer.json` file.
+
+### Basic entities
+
+The "[basic entities](https://docs.contao.org/manual/en/article-management/insert-tags/#basic-entities)" such as
+`[-]`, `[nbsp]` etc. are not automatically converted back during rendering. Instead they are converted in the back
```

Comment:
Frankly, I don't get the explanation 🙈  Where exactly are the entities not converted back? And how does this confuse people so they would check if there is an entry in the `UPGRADE.md` file? And how is the new behavior different from the previous one?

Reply:
Previously, Contao's "basic entities", e.g. `[-]`, `[nbsp]` etc. where handled like insert tags - i.e. when the `fe_page.html5` template was rendered, they were converted back to `&shy;`, `&nbsp;` etc. This is now not the case anymore, they will directly be transformed to their HTML counterparts when saving a DCA record in the back end, for each field where the conversion is enabled in the DCA `eval`.

Reply:
And why do we have to mention this in the `UPGRADE.md` file? What do the users have to do?

Reply:
They need to set `'basicEntities' => true` in the fields where they use these basic entities.